python/mxnet/_numpy_op_doc.py

```diff
@@ -223,3 +223,46 @@ def _np_dot(a, b, out=None):
     array(29884.)
     """
     pass
+
+
+def _np_transpose(a, axes=None):
+    """
+    transpose(a, axes=None)
+
+    Permute the dimensions of an array.
+
+    Parameters
+    ----------
+    a : ndarray
+        Input array.
+    axes : list of ints, optional
+        By default, reverse the dimensions,
+        otherwise permute the axes according to the values given.
+
+    Returns
+    -------
+    p : ndarray
+        a with its axes permuted.
+
+    Notes
+    -----
+    This function differs from the original `numpy.transpose
+    <https://docs.scipy.org/doc/numpy/reference/generated/numpy.transpose.html>`_ in
+    the following way(s):
+
+    - only ndarray is accepted as valid input, python iterables are not supported
+
+    Examples
+    --------
+    >>> x = np.arange(4).reshape((2,2))
+    >>> x
+    array([[0., 1.],
```

Comment:
dtype=float32 is the default type and will not appear in the up to date version.

Reply:
> dtype=float32 is the default type and will not appear in the up to date version.

Thank you for reminding. Will fix that.